fuzzaldrin-plus/index.d.ts

```diff
@@ -5,25 +5,32 @@
 
 export as namespace fuzzaldrin;
 
-export interface IQueryOptions {
+export interface QueryOptions {
```

Comment:
@mdahamiwal that's not quite accurate. types-2.0 types are published to @types and are meant to be used by the TypeScript 2.0 compiler. So unfortunately yes, you do need to worry about backwards compatibility. See the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md) for clarification.